algorithms/CSharp/src/Strings/character-limit.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Algorithms.Strings{
+
+    public class TextHelper{
+        public static void Main()
+        {
+            Console.WriteLine("Please enter a string");
+            string input = Console.ReadLine();
+            Console.WriteLine("Please enter the number of characters you want to show");
+            int num = Convert.ToInt32(Console.ReadLine());
```

Comment:
This will throw an exception if someone enters more than 1 integer. For example, someone enters "5 7 9"